src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,47 @@
 package core.basesyntax;
 
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final String COMMA_SEPARATOR = ",";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        try {
+            writeDataToFile(calculateData(readDataFromFile(fromFileName)),toFileName);
+        } catch (IOException e) {
+            throw new RuntimeException("Error while working with files");
```

Comment:
in this case it will be difficult to find why error was thrown, because it isn't informative enough 
but we can do something like `throw new RuntimeException("...", e)`, so stack trace will be more informative